### PR TITLE
Glide improvements

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,15 +4,14 @@ PYTHON_ENV=${BUILD_DIR}/python-env/
 
 .PHONY: test
 test: python-env
-	mkdir -p build/src
+	mkdir -p build/src/beatpath
```
Comment (urso) [inline Makefile]: build/src/beatpath correct?

Comment (ruflin) [inline Makefile]: I think so. This correlates with the config here: https://github.com/elastic/beat-generator/pull/10/files#diff-4a2ac964905a39da7175b4ff878c014cR5

Comment (urso) [inline Makefile]: then I do not understand why this get's hardcoded to `beatpath`. Why not cookiecutter.beat_path ?

Comment (ruflin) [inline Makefile]: This is for testing only, not the Makefile of the generated template.

**Pull request merged**